### DISCUSSION TITLE
Diff toolbar button for overriden shadows?

### DISCUSSION THREAD

**Discussion by saqimtiaz** [TiddlyWiki/TiddlyWiki5#6058] [Q&A] [answered]
I have a vague memory of a toolbar command or similar for showing a diff between the current version of a tiddler and its underlying shadow.

Does such a thing exist in the core, or am I perhaps recalling something I or someone else prototyped at some point?

**Comment by Jermolene** [comment, marked as answer]
It's one of the core preview options:

<img width="1219" alt="image" src="https://user-images.githubusercontent.com/174761/133990539-0a56b2e4-4a6b-4b19-9265-d11b4836b22b.png">

**Reply by saqimtiaz** [reply]
Excellent, thank you.
